extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -234,6 +245,23 @@ export class SubscriptionService {
 
       user().assert(this.pageConfig_, 'Page config is null');
 
+      if (this.doesViewerProvideAuth_) {
+        const serviceIds = ['local'];
+        this.platformStore_ = new PlatformStore(serviceIds);
+        this.viewer_.sendMessageAwaitResponse('auth', dict())
```

Comment:
There must be a request including `publicationId` and `productId`. Classically we also always include `origin`.